eng/test-job.yml

```diff
@@ -179,6 +181,12 @@ jobs:
           # DotNet-HelixApi-Access variable group
           helixAccessToken: $(HelixApiAccessToken)
 
+        # Sets up the template to run the corefx tests
+        ${{ if eq(parameters.corefxTests, 'true') }}:
+          helixProjectFile: 'eng/helixcorefxtests.proj /t:Test /p:ArcadeBuild=True'
```

Comment:
Could you make this the project's `DefaultTarget`? I'd prefer not to name it `helixProjectFile` if it contains anything other than a file path.

Reply:
I'll rename it helixProjectArguments or something else.

Reply:
I've tried setting DefaultTargets="Test" and it didn't work.

Reply:
Weird... too bad.